Fix Link to Policy Docs (#200)
hashicorp/terraform-docs-common: website/docs/cloud-docs/run/cli.mdx

> The [CLI integration](/cli/cloud) brings Terraform Cloud's collaboration features into the familiar Terraform CLI workflow. It offers the best of both worlds to developers who are already comfortable with using the Terraform CLI, and it can work with existing CI/CD pipelines.

Reviewer: ```suggestion
The [CLI integration](/cli/cloud) brings Terraform Cloud's collaboration features into the familiar Terraform CLI workflow. It stronger functionality developers who are already comfortable with using the Terraform CLI and works with existing CI/CD pipelines.
```

Maybe just a personal thing, but saying "It offers the best of both worlds" feels like marketing speak / overly colloquial language to me. Feel free to ignore if you disagree.

Author: this is outside the scope of this PR, so I think I'm going to ignore for now :)

> You can start runs with the standard `terraform plan` and `terraform apply` commands and then watch the progress of the run from your terminal. These runs execute remotely in Terraform Cloud; they use variables from the appropriate workspace, enforce any applicable [Sentinel or OPA policies][/cloud-docs/policy-enforcement], and can access Terraform Cloud's [private registry][private] and remote state inputs.

Reviewer: ```suggestion
You can start runs with the standard `terraform plan` and `terraform apply` commands and then watch the progress of the run from your terminal. These runs execute remotely in Terraform Cloud, use variables from the appropriate workspace, enforce any applicable [Sentinel or OPA policies][/cloud-docs/policy-enforcement], and can access Terraform Cloud's [private registry][private] and remote state inputs.
```

Suggest removing semicolon and forming 1 sentence. If 1 sentence is too long / a run-on, suggest splitting into 2 sentences.
